src/features/hotspots/setup/HotspotSelectionListItem.tsx

```diff
@@ -12,6 +12,7 @@ import NEBRAIN from '../../../assets/images/nebra-in.svg'
 import NEBRAOUT from '../../../assets/images/nebra-out.svg'
 import BOBCAT from '../../../assets/images/bobcat.svg'
 import SYNCROBIT from '../../../assets/images/syncrobit.svg'
+import LONGAPONE from  '../../../assets/images/longapone.svg'
```

Comment:
This should to be `longap-one.svg`

Reply:
Also can you remove the extra space after from

Reply:
Fixed